src/metabase/models/humanization.clj

```diff
@@ -47,7 +48,7 @@
 ;; simple replaces hyphens and underscores with spaces and capitalizes
 (defmethod name->human-readable-name :simple
   ([s] (name->human-readable-name :simple s))
-  ([_, ^String s]
+  ([_ ^String s]
```

Comment:
Maybe change these `s` variable names to something else now that schema is being aliased as `s`?

Reply:
If schema is aliased as `s` it's fine, the alias only applies to referring to things from that namespace (e.g. `s/Keyword` or `s/Int`. `s` by itself wouldn't refer to `schema.core`, so it's not overshadowing anything